02-express-tutorial/routes/people.js

```diff
@@ -0,0 +1,30 @@
+const express = require("express");
+const router = express.Router();
+
+let { people } = require("../data"); //from data.js file
+
+const {
+  getPerson,
+  getPeople,
+  createPerson,
+  createPersonPostman,
+  updatePerson,
+  deletePerson,
+} = require("../controllers/people");
+
+//Using the router.method code below is fine or you can use less lines doing the
+//  exact same thing with the router.route code
+
+//router.get("/", getPeople);
+//router.post("/", createPeople);
+//router.post("/postman", createPersonPostman);
+//router.put("/:id", updatePerson);
+//router.delete("/:id", deletePerson);
```

Comment:
Nit: Remove commented out code before submitting PRs.  This is good practice because when you start working in a team environment this will clutter up the main branch and make things more confusing